services/rsources/handler.go

```diff
@@ -488,6 +488,11 @@ func (sh *sourcesHandler) init() error {
 	}
 
 	if sh.sharedDB != nil {
+		if err := sh.sharedDB.Ping(); err != nil {
```

Comment:
We shouldn't ignore errors in case we are in processor, so whether it will fail or not, should be configurable